### PR TITLE
FEATURE: TMA-1165 Create Help brick and make it default entry point in Docker image

### PR DESCRIPTION
**Pull request by sadam21**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/814/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1628/ : FAILURE in 1m 27s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1889/ : SUCCESS in 1m 15s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/68/ : SUCCESS in 7m 49s

File: lib/gooddata/lcm/actions/help.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        BRICK_PIPELINE_SUFFIX = /_brick_pipeline$/
+
+        def create_help
+          "Available bricks:" +
+            GoodData::Bricks::Pipeline.methods
+              .grep(BRICK_PIPELINE_SUFFIX)
+              .map { |method| "\n\t" + method.to_s.sub(BRICK_PIPELINE_SUFFIX, '') }.join
+        end
+
+        def call(params)
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/help.rb]: info: Has unused parameter 'params' (UnusedParameters) (reek)

Comment (yenkins) [inline lib/gooddata/lcm/actions/help.rb]: warning: Lint/UnusedMethodArgument: Unused method argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used. You can also write as `call(*)` if you want the method to accept any arguments but don't care about them. (rubocop)

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/816/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1630/ : SUCCESS in 1m 14s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1891/ : SUCCESS in 1m 17s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/70/ : SUCCESS in 7m 28s

File: lib/gooddata/bricks/help_brick.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require_relative 'brick'
+
+module GoodData
+  module Bricks
+    # Simple brick used for printing help message anout bricks
```
Comment (jimirocks) [inline lib/gooddata/bricks/help_brick.rb]: typo? `anout`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/818/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1632/ : SUCCESS in 1m 21s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1893/ : SUCCESS in 1m 23s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/72/ : SUCCESS in 7m 40s

File: lib/gooddata/bricks/help_brick.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    class HelpBrick < GoodData::Bricks::Brick
+      def version
+        '0.0.1'
+      end
+
+      # HelpWorld brick entry-point
+      def call(params)
```
Comment (yenkins) [inline lib/gooddata/bricks/help_brick.rb]: info: Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction) (reek)

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/824/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1638/ : SUCCESS in 1m 13s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1899/ : SUCCESS in 1m 17s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/77/ : SUCCESS in 8m 17s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: spec/lcm/integration/help_spec.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+require 'gooddata'
+
+describe 'HelpBrick' do
+  it "print help" do
+    @brick_result = GoodData::Bricks::Pipeline.help_brick_pipeline.call({})
+    expect(@brick_result[:results]["Help"].map { | brick | brick[:available_brick] }).to eq(
```
Comment (yenkins) [inline spec/lcm/integration/help_spec.rb]: warning: Layout/SpaceAroundBlockParameters: Space before first block parameter detected. (rubocop)

Comment (yenkins) [inline spec/lcm/integration/help_spec.rb]: warning: Layout/SpaceAroundBlockParameters: Space after last block parameter detected. (rubocop)

File: spec/lcm/integration/help_spec.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+      %w[ release users rollout provisioning user_filters help hello_world ]
```
Comment (yenkins) [inline spec/lcm/integration/help_spec.rb]: warning: Layout/SpaceInsidePercentLiteralDelimiters: Do not use spaces inside percent literal delimiters. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/830/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1643/ : FAILURE in 1m 34s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1904/ : SUCCESS in 1m 15s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/82/ : SUCCESS in 7m 34s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/831/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1644/ : SUCCESS in 2m 47s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1905/ : SUCCESS in 2m 38s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/83/ : SUCCESS in 6m 48s

**Comment by sadam21** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/832/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1645/ : SUCCESS in 2m 35s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1906/ : SUCCESS in 2m 39s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/84/ : SUCCESS in 7m 27s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/133/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1646/ : SUCCESS in 1m 21s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1907/ : SUCCESS in 1m 14s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1555/ : FAILURE in 10s
- gooddata-ruby-lcm-integration-tests-zuul : SKIPPED
- gooddata-ruby-lcm-pipeline : SKIPPED

**Comment by jimirocks** [comment]
Merging, as the relevant tests passed.

**Pull request merged**